pkg/bindings/test/common_test.go

```diff
@@ -178,11 +178,19 @@ func (b *bindingTest) cleanup() {
 func (b *bindingTest) Pull(name string) {
 	p := b.runPodman([]string{"pull", name})
 	p.Wait(45)
+	if p.ExitCode() != 0 {
+		fmt.Printf("Could not pull %s, aborting\n", name)
+		os.Exit(1)
```

Comment:
exit(1) is not an appropriate error handling when running under ginkgo.
Please use the normal Expect().To() flow. Ginkgo will now that it failed in the setup and will not run the suite.

Reply:
sure just import what you need.
I know a lot of existing tests use os.Exit() but this is really bad, ginkgo cannot provide proper output in such cases. Especially if you keep ginkgo v2 in mind where it buffers the output when run in parallel so if we just exit early good luck debugging that.